lib/push.js

```diff
@@ -10,6 +10,11 @@ var ERR_DATA_BUT_NO_KEYS = 'Data payload present but missing key(s)'
 
 var LOG_OP_PUSH_TO_DEVICES = 'push.pushToDevices'
 
+var PUSH_PAYLOAD_SCHEMA_VERSION = 1
+var pushCommands = {
```

Comment:
nit: maybe make this `pushCommands` all uppercase to signal `const` without using it, to follow the content-server and other variables above.